app/_hub/kong-inc/graphql-rate-limiting-advanced/overview/_index.md

```diff
@@ -19,6 +19,11 @@ cost for a given time window.
   * Redis configuration values are ignored if the `cluster` strategy is used.
   * PostgreSQL 9.5+ is required when using the `cluster` strategy with `postgres` as the backing Kong cluster datastore.
   * The `dictionary_name` directive was added to prevent the usage of the `kong` shared dictionary, which could lead to `no memory` errors.
+  * Introspection endpoint is generated based on the Kong service path, hence service path should be defined with actual path instead of appending from the route path.
```

Comment:
```suggestion
  * The introspection endpoint is generated based on the Kong service path, so the service path should be defined with an actual path instead of appending from the route path.
```
Using "so" instead of "hence" for American english phrasing (we use American english as the basis for our docs).